internal/pkg/describe/env.go

```diff
@@ -50,23 +65,68 @@ func NewEnvDescriber(projectName string, envName string) (*EnvDescriber, error)
 		return nil, fmt.Errorf("assuming role for environment %s: %w", env.ManagerRoleARN, err)
 	}
 	return &EnvDescriber{
-		env:          env,
-		store:        svc,
-		apps:         apps,
-		sessProvider: sess,
+		env:      env,
+		store:    svc,
+		proj:     proj,
+		apps:     apps,
+		rgClient: resourcegroups.New(sess),
 	}, nil
 }
 
 // Describe returns info about a project's environment.
 func (e *EnvDescriber) Describe() (*EnvDescription, error) {
-	var tags map[string]string
+	appsForEnv, err := e.filterAppsForEnv()
+	if err != nil {
+		return nil, err
+	}
+
 	return &EnvDescription{
 		Environment:  e.env,
-		Applications: e.apps,
-		Tags:         tags,
+		Applications: appsForEnv,
+		Tags:         e.proj.Tags,
```

Comment:
I thought `Tags` field should be tags for the environment stack instead of project's tags? What tags do we want to show in `env show`?

Reply:
We will show both. I am adding the tags from the environment stack next! :)